WORKSPACE

```diff
@@ -109,10 +109,9 @@ distdir_tar(
     archives = [
         "e0b0291b2c51fbe5a7cfa14473a1ae850f94f021.zip",
         "f83cb8dd6f5658bc574ccd873e25197055265d1c.tar.gz",
-        "java_tools_pkg-0.5.1.tar.gz",
```

Comment:
Why is this gone?

Reply:
This was leftover by mistake from an older change. It's an old version of java_tools.

Reply:
I'd remove it in a separate change so as to decouple the risks of these two changes, but your call.